src/q7/instrusensors-ELEC2811/exam/2019/Janvier/All/instrusensors-ELEC2811-exam-2019-Janvier-All.tex

```diff
@@ -0,0 +1,246 @@
+
+\hypertitle{Instrumentation and Sensors}{7}{ELEC}{2811}{2019}{Janvier}{All}
+{Alice Borbath \and Martin Braquet \and Olivier Leblanc}
+{David Bol and Laurent Francis}
+
+Evaluation marks:
+\begin{itemize}
+    \item Two reports: 4/20
+    \item Final report: 8/20
+    \item Exam: 5/20 for the 14 questions and 3/20 for the open question about the project
+\end{itemize}
+
+
+\section{}
+
+For each of the following sensors, choose if it is a direct or an indirect sensor:
+
+\begin{itemize}
+    \item A posistor
+    \item A piezoresistor
+    \item A capacitive accelerometer
+    \item A chemical based hygristor
+    \item 
+\end{itemize}
+
+\nosolution
+
+\section{}
+
+As regards a piezoresistance with a strain gauge, select the \textbf{false} proposition:
+
+\begin{itemize}
+    \item The strain gauge factor of the silicium highly depends on its doping.
+    \item The strain gauge factor for metals can vary over larges values between $-120$ and 200.
+    \item The strain gauge factor depends on the Poisson's coefficients and the relative variation of the resistivity with the stress.
+    \item The strain gauge factor measures the ratio between the relative variation of the resistance and the strain.
+    \item 
+\end{itemize}
+
+
+\nosolution
+
+\section{}
+
+Give four oscillators and their type (sine or binary).
+
+
+\nosolution
+
+\section{}
+
+Give three applications based on the piezoelectric effect.
+
+
+\nosolution
+
+\section{}
+
+We consider the general configuration of a Sallen-Key filter.
+
+\begin{figure}[H]
+    \centering
+    \includegraphics[width=0.5\textwidth]{sallen.png}
+\end{figure}
+
+For each statement, select true or false: 100 \% for a right answer, 0 \% for no answer and $-50$ \% for a wrong answer.
+
+\begin{itemize}
+    \item When $Z_1$ and $Z_2$ are capacitors and $Z_3$ and $Z_4$ are resistors, it is a high pass filter.
+    \item The gain depends on the impedances $Z_1$ to $Z_4$.
+    \item This filter is always used for sensors with a changing resistance.
+    \item The Sallen-key filter is considered as a second order filter.
+\end{itemize}
+
+\nosolution
+
+
+\section{}
+
+As regards capacitive sensors, select the \textbf{false} proposition:
+
+\begin{itemize}
+    \item The relation between the capacitance and the stimulus is always linear in a capacitor with parallel plates.
+    \item The measurement of the water level in a tank with a variable capacitor must come with a temperature measurement.
+    \item A capacitor has a linear relation with humidity.
+    \item 
+    \item 
+\end{itemize}
+
+\nosolution
+
+
+\section{}
+
+A team of engineers is asked to measure leakages of hydrogen in order to estimate the risk of explosion. Leakage of hydrogen becomes dangerous when the concentration  in air reaches 4\% since it associates with oxygen and it results in an explosion.  The hydrogen is stored in a room where other bottles of gases can be placed, mostly monoxide carbon and ammonia. 
```

Comment:
```suggestion
A team of engineers is asked to measure leakages of hydrogen in order to estimate the risk of explosion. Leakage of hydrogen becomes dangerous when the concentration in air reaches 4 \% since it associates with oxygen and it results in an explosion.  The hydrogen is stored in a room where other bottles of gases can be placed, mostly carbon monoxide and ammonia. 
```